path_test.go

```diff
@@ -12,14 +12,14 @@ import (
 )
 
 const (
-	routeBlindingTestFileName            = "testdata/route-blinding-test.json"
-	onionRouteBlindingTestFileName       = "testdata/onion-route-blinding-test.json"
-	blindedOnionMessageOnionTestFileName = "testdata/blinded-onion-message-onion-test.json"
+	routeBlindingTestFileName            = "testdata/route-blinding-test.json"              //nolint:lll
```

Comment:
nit: we can put `nolint:lll` on the top of `const (` so we don't need to declare it multiple times.